yang/frr-ospf-route-map.yang

```diff
@@ -0,0 +1,65 @@
+module frr-ospf-route-map {
+  yang-version 1.1;
+  namespace "http://frrouting.org/yang/ospf-route-map";
+  prefix frr-ospf-route-map;
+
+  import ietf-inet-types {
+    prefix inet;
+  }
+
+  import frr-route-map {
+    prefix frr-route-map;
+  }
+
+  organization
+    "Free Range Routing";
+  contact
+    "FRR Users List:       <mailto:frog@lists.frrouting.org>
+     FRR Development List: <mailto:dev@lists.frrouting.org>";
+  description
+    "This module defines bgp route map settings";
```

Comment:
1. `bgp` -> `ospf`
2. Should we have a separate yang module for ospf6? ospf6 implements `forwarding-address` while ospf does not.

Reply:
Done